### PR TITLE
lightning: update the default value of add-index-by-sql to false

### PR DESCRIPTION
**Pull request by hfxsd**
### What is changed, added or deleted? (Required)

as tittle

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [x] master (the latest development version)
- [x] v7.0 (TiDB 7.0 versions)
- [ ] v6.6 (TiDB 6.6 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.4 (TiDB 6.4 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:https://github.com/pingcap/docs-cn/pull/13515
- Other reference link(s):https://github.com/pingcap/docs/pull/12948

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Review by gozssky — APPROVED** [review]

File: tidb-lightning/tidb-lightning-configuration.md [hunk 1]
```diff
@@ -164,7 +164,7 @@ addr = "172.16.31.10:8287"
 # The default value is `MaxInt64` bytes, that is, 9223372036854775807 bytes.
 # disk-quota = "10GB"
 
-# Specifies whether Physical Import Mode adds indexes via SQL. The default value is automatically decided based on the TiDB version. If the TiDB version is earlier than v7.0.0, the default value is `false`, which means that TiDB Lightning will encode both row data and index data into KV pairs and import them into TiKV together. This mechanism is consistent with that of the historical versions. Starting from TiDB v7.0.0, the default value is `true`, which means that TiDB Lightning adds indexes via SQL after importing the row data.
+# Specifies whether Physical Import Mode adds indexes via SQL. The default value is `false`, which means that TiDB Lightning will encode both row data and index data into KV pairs and import them into TiKV together. If you set it to `true`, it means that TiDB Lightning adds indexes via SQL after importing the row data.
```
Comment (hfxsd) [inline tidb-lightning/tidb-lightning-configuration.md:167]:
```suggestion
# Specifies whether Physical Import Mode adds indexes via SQL. The default value is `false`, which means that TiDB Lightning will encode both row data and index data into KV pairs and import them into TiKV together. This mechanism is consistent with that of the historical versions. If you set it to `true`, it means that TiDB Lightning adds indexes via SQL after importing the row data.
```

File: tidb-lightning/tidb-lightning-physical-import-mode-usage.md [hunk 2]
```diff
@@ -41,6 +41,10 @@ sorted-kv-dir = "./some-dir"
 # node in the physical import mode. 0 by default, which means no limit.
 # store-write-bwlimit = "128MiB"
 
+# Specifies whether Physical Import Mode adds indexes via SQL. The default value is `false`, which means that TiDB Lightning will encode both row data and index data into KV pairs and import them into TiKV together. If you set it to `true`, it means that TiDB Lightning adds indexes via SQL after importing the row data.
```
Comment (hfxsd) [inline tidb-lightning/tidb-lightning-physical-import-mode-usage.md:44]:
```suggestion
# Specifies whether Physical Import Mode adds indexes via SQL. The default value is `false`, which means that TiDB Lightning will encode both row data and index data into KV pairs and import them into TiKV together. This mechanism is consistent with that of the historical versions. If you set it to `true`, it means that TiDB Lightning adds indexes via SQL after importing the row data.
```

File: tidb-lightning/tidb-lightning-configuration.md [hunk 3]
```diff
@@ -164,7 +164,7 @@ addr = "172.16.31.10:8287"
...
+# Specifies whether Physical Import Mode adds indexes via SQL. The default value is `false`, which means that TiDB Lightning will encode both row data and index data into KV pairs and import them into TiKV together. This mechanism is consistent with that of the historical versions. If you set it to `true`, it means that TiDB Lightning adds indexes via SQL after importing the row data.
 # The benefit of adding indexes via SQL is that you can separately import data and import indexes, and import data more quickly. After the data is imported, even if the indexes fail to be added, it does not affect the consistency of the imported data.
 # add-index-by-sql = true
```
Comment (hfxsd) [inline tidb-lightning/tidb-lightning-configuration.md:170]:
```suggestion

# 在使用 TiDB Lightning 导入多租户的 TiDB cluster 的场景下，指定对应的 key space 名称。默认取值为空字符串，表示 TiDB Lightning 会自动获取导入对应租户的 key space 名称；如果指定了值，则使用指定的 key space 名称来导入。
When you use TiDB Lightning to import a multi-tenant TiDB cluster, use this parameter to specify the corresponding key space name. The default value is an empty string, which means TiDB Lightning will automatically get the key space name of the corresponding tenant to import. If you specify a value, the specified key space name will be used to import.
# keyspace-name = ""
```

**Review by Frank945946 — APPROVED** [review]

File: tidb-lightning/tidb-lightning-configuration.md [hunk 4]
```diff
@@ -164,10 +164,12 @@ addr = "172.16.31.10:8287"
...
```
Comment (qiancai) [inline tidb-lightning/tidb-lightning-configuration.md:169]: 中英文这里是否需要修改为 # add-index-by-sql = false

File: tidb-lightning/tidb-lightning-physical-import-mode-usage.md [hunk 5]
```diff
@@ -41,6 +41,10 @@ sorted-kv-dir = "./some-dir"
 # node in the physical import mode. 0 by default, which means no limit.
 # store-write-bwlimit = "128MiB"
 
+# Specifies whether Physical Import Mode adds indexes via SQL. The default value is `false`, which means that TiDB Lightning will encode both row data and index data into KV pairs and import them into TiKV together. This mechanism is consistent with that of the historical versions. If you set it to `true`, it means that TiDB Lightning adds indexes via SQL after importing the row data.
+# The benefit of adding indexes via SQL is that you can separately import data and import indexes, and import data more quickly. After the data is imported, even if the indexes fail to be added, it does not affect the consistency of the imported data.
+# add-index-by-sql = true
```
Comment (qiancai) [inline tidb-lightning/tidb-lightning-physical-import-mode-usage.md:46]: 中英文这里是否需要修改为 # add-index-by-sql = false

**Review by qiancai — APPROVED** [review]
other LGTM

File: tidb-lightning/tidb-lightning-physical-import-mode-usage.md (hunk 5 above)
Comment (hfxsd) [inline tidb-lightning/tidb-lightning-physical-import-mode-usage.md:46]:
```suggestion
# add-index-by-sql = false
```

Comment (hfxsd) [reply tidb-lightning/tidb-lightning-physical-import-mode-usage.md:46]: ok

File: tidb-lightning/tidb-lightning-configuration.md (hunk 4 above)
Comment (hfxsd) [inline tidb-lightning/tidb-lightning-configuration.md:169]:
```suggestion
# add-index-by-sql = false
```

Comment (hfxsd) [reply tidb-lightning/tidb-lightning-configuration.md:169]: ok updated

**Pull request merged**